frontend/src/pages/ActivityReport/index.js

```diff
@@ -539,6 +540,7 @@ function ActivityReport({
             endDate: endDateToSave,
             regionId: formData.regionId,
             approverUserIds: approverIds,
+            version: 2,
```

Comment:
Isn't this path for a new report? Why do we set version to 2?

Reply:
All reports saved after this change goes through will be version 2 reports

Reply:
Yea that's my bad thought we were incrementing the value.